### PR TITLE
Bugfix/determine client version

### PR DESCRIPTION
**Pull request by samface**
These commits resolves two critical issues:

1. Composer.lock file not in the root path of this composer package because it has been installed as part of another composer package. Examples of this are the Magento2 modules. The solution I have here is traversing up the file path until a composer.lock file is found. In case this wouldn't be enough, we need to either revert the commit removing the version from composer.json or maybe add a CHANGELOG.md to parse the current version from.

2. VERSION as a constant or environment variable name is too generic and has a high risk of causing conflicts. This is fixed by using the __NAMESPACE__ of the class as scope/prefix.

Another improvement is no longer needing our own custom file path resolver. It was a workaround for a bug in PHP 5.3.x that never was relevant for this repository since it's using and requiring 7.x coding standards anyway.

### CODE REVIEW

**Comment by asbjornu** [comment]
Thanks @samface. Do you think you could add `sammy.nordstrom@packoplock.se` to your GitHub account so GitHub can recognise your commits and also [sign our CLA](https://cla-assistant.io/SwedbankPay/swedbank-pay-sdk-php?pullRequest=28)?

**Comment by samface** [comment]
@asbjornu I've done what I can to resolve the CI-issues now. It remains a problem that there are no environment variables set for merchant token and payee id. There also seems to be issues with the integration tests that I assume someone is looking into?

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5624753&bid=18425270)

File: tests/bootstrap.php [hunk 1]
```diff
@@ -31,10 +33,15 @@
     define('MERCHANT_TOKEN', $config['merchant_token']);
     define('PAYEE_ID', $config['payee_id']);
 
-    $data = json_decode(file_get_contents(__DIR__ . '../composer.json'), true);
+    $data = json_decode(file_get_contents(__DIR__ . '/../composer.json'), true);
```
Comment (payex-dev) [inline tests/bootstrap.php:36]: Codacy Issue found: [The use of function file_get_contents() is discouraged](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5624753)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5624753&bid=18425270)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5624753&bid=18425270)

**Pull request merged**